prepare_offline_installation/prepare_offline_installation.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# simplify your orcharhino Server offline installation by generating commands based on a list of repositories
+
+###################################################################
+# adapt path to your offline medium, for example an external disk
+repo_path = "/media/my_disk/my_repos/"
+# adapt list of repositories
+repositories = [
+    "Insert list from ATIX Service Portal:",
+    "https://atixservice.zendesk.com/hc/de/articles/4414413251346",
+]
+###################################################################
+
+# create repo file
+repo_file = "/tmp/orcharhino.repo"
+repo_file_content = """"""
+skeleton = """
+[REPO_ID]
+name=REPO_ID
+baseurl=file://REPO_PATH
+enabled=1
+gpgcheck=0
+"""
+for repository in repositories:
+    repo_file_content = repo_file_content + skeleton.replace("REPO_ID", repository).replace("REPO_PATH", repo_path + repository)
+with open(repo_file, "w") as f:
+    f.write(repo_file_content)
+
+# create bash script to enable repositories
+enable_repos_content = """"""
+enable_repos_file = "/tmp/enable_repositories.sh"
+for repository in repositories:
+    enable_repos_content = enable_repos_content + "subscription-manager repos --enable=" + repository + "\n"
+with open(enable_repos_file, "w") as f:
+    f.write("#!/bin/bash\n\nset -e\n\n")
```

Comment:
Might be more readable using a string with newlines in it `"""`
Given that this is not the only usage of this string, putting it into a variable might be a good idea (e.g. `bash_preamble`)

Reply:
Applied.